intake-catalogs/ocean.yaml

```diff
@@ -77,6 +77,21 @@ sources:
       storage_options:
         token: anon
       consolidated: True
+      
+   ECCO_layers:
```

Comment:
@cspencerjones  - I think the indentation here is wrong. That's why the CI is failing:

https://travis-ci.org/pangeo-data/pangeo-datastore/builds/572511324#L635